src/components/Header/components/LoginButton.tsx

```diff
@@ -64,7 +64,7 @@ const LoginButton = chakra(({ isMobile = false }: LoginButtonProps) => {
           />
         </Button>
         {isOpen && (
-          <HeaderComponents.Login
+          <Login
```

Comment:
I understand that you need to remove `HeaderComponents.`, but I now worry that the component names are too generic.  How would you feel about changing the component names to include `Header` as part of the full component name?  `HeaderLoginButton`, `HeaderUpperNav`, etc.

I’m thinking about this because we’re starting to work non-atomic level components and we could potentially see other components with similar names.

Reply:
Yeah, I'll update this PR with that change. The namespacing, unfortunately, didn't work as well because of the circular dependencies.